blockchain/types/block.go

```diff
@@ -41,6 +42,8 @@ const (
 )
 
 var (
+	// EmptyRootHash is transaction/receipt root hash when there is no transaction.
+	// This value is initialized in InitDeriveSha.
 	EmptyRootHash = common.Hash{}
```

Comment:
Do you know why this value is different with geth below?
```
	EmptyRootHash  = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
```

Reply:
@jeongkyun-oh Refer https://github.com/ground-x/klaytn/commit/9b6ebaeb3a084e1bbdc3888f2ed56e464f7231a3. :)

I think we change it and this code is ok. :)